charts/sonarqube-dce/README.md

```diff
@@ -415,7 +415,9 @@ The following table lists the configurable parameters of the SonarQube chart and
 | `jdbcOverwrite.jdbcSecretName`        | Alternatively, use a pre-existing k8s secret containing the DB password                                                                                      | `None`                                                        |
 | `jdbcOverwrite.jdbcSecretPasswordKey` | If the pre-existing k8s secret is used this allows the user to overwrite the 'key' of the password property in the secret                                    | `None`                                                        |
 
-### Bundled Postgres Chart
+### Bundled Postgres Chart (DEPRECATED)
```

Comment:
Can we take the chance to fix the typo here (and elsewhere).  `Postgres` -> ` PostgreSQL`?

Reply:
That is not really a typo: https://www.postgresql.org/docs/current/history.html#HISTORY-POSTGRESQL. :)

But I also prefer PostgreSQL. :)